.github/workflows/mariadb.yml

```
@@ -48,6 +48,19 @@ jobs:
         with:
           path: mariadb/storage/mroonga
           repository: mroonga/mroonga
+      - name: Install ccache
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ccache
+      - name: Prepare ccache
+        run: |
+          echo "CCACHE_DIR=${PWD}/mariadb/ccache" >> ${GITHUB_ENV}
```

How about avoiding using source directory?

```suggestion
          echo "CCACHE_DIR=${PWD}/ccache" >> ${GITHUB_ENV}
```

---

fix: [6447040](https://github.com/mroonga/mroonga/pull/901/commits/6447040dd90af9791a7c4f1b69a4754c91fe153c) I configured it out of source.